### DISCUSSION TITLE
SO all docker containers don´t start after 1 month uptime

### DISCUSSION THREAD

**Discussion by Ax386** [Security-Onion-Solutions/securityonion#3394] [Unsupported Versions] [answered]
Hi,
after more than 1 month uptime, all dockers dont start anymore.. message during boot initialization: Cannot write to process directory. Do you have permissions to write to /var/cache/salt/minion/proc/20210311121354539884

..if i try to make a new directory on /var/cache/salt/minion/proc ... message: mkdir: cannot create directory ‘20210311121354539884’: No space left on device

i run SO as a standalone deployment...

any ideas?

thanks a lot,
Alex

**Comment by dougburks** [comment, marked as answer]
Looks like you're out of disk space.

Check to make sure your box meets the hardware requirements:
https://docs.securityonion.net/en/2.3/hardware.html
https://docs.securityonion.net/en/2.3/partitioning.html

If your box does meet the hardware requirements, then try to figure out what's using the space.

**Comment by Ax386** [comment]
Hi Doug,

thanks for your fast answer! ...right,...dockers allocated all ressources of diskspace.... new installation 2.3.30 fixed it ;-) everything works fine
thx,
Alex
